rules/experimental/rbe/exec_properties.bzl

```diff
@@ -156,3 +242,105 @@ def merge_dicts(*dict_args):
     for dictionary in dict_args:
         result.update(dictionary)
     return result
+
+def _exec_property_sets_repository_impl(repository_ctx):
+    repository_ctx.file(
+        "BUILD",
+        content = "",
+        executable = False,
+    )
+    repository_ctx.file(
+        "exec_propert_sets.bzl",
+        content = repository_ctx.attr.exec_property_sets_content,
+        executable = False,
+    )
+
+# _exec_property_sets_repository is a repository rule that creates a repo with the specified exec_property_sets.
+_exec_property_sets_repository = repository_rule(
+    implementation = _exec_property_sets_repository_impl,
+    local = True,
+    attrs = {
+        "exec_property_sets_content": attr.string(mandatory = True),
```

Comment:
add a doc attribute to exec_property_sets_content and explain what its for here

Reply:
done